src/components/Rainbowcrack/Rainbowcrack.tsx

```diff
@@ -32,17 +32,26 @@ const RainbowCrack = () => {
     const [hasSaved, setHasSaved] = useState(false); // State variable to track whether output has been saved
 
     // Component Constants
-    const title = "RainbowCrack"; // Title of the component.
+    const title = "RainbowCrack"; // Title of the component
+
     const description =
-        "RainbowCrack is a computer program which utilises rainbow tables to be used in password cracking."; // Description of the component.
+        "RainbowCrack is a command-line tool that uses rainbow tables to crack password hashes. It supports multiple hash algorithms, including LM, NTLM, MD5, SHA1 and SHA256."; // Description of the component.
+
     const steps =
-        "How to use RainbowCrack \n" +
-        "Step 1: Enter a hash value. (E.g. 5d41402abc4b2a76b9719d911017c592) \n" +
-        "Step 2: Simply tap on the crack button to crack the hash key. \n" +
-        "The user can even save the output to a file by assigning a file-name under 'save output to file' option."; // Steps to use the component.
-    const sourceLink = ""; // Link to the source code (or RainbowCrack documentation).
+        "How to use RainbowCrack:\n" +
+        "1. Ensure that your rainbow tables (*.rt, *.rtc) are stored in a directory.\n" +
+        "2. Enter the hash value you want to crack in the input field. For a single hash, use the format: ./rcrack [path to tables] -h [hash].\n" +
+        "   - Example: ./rcrack . -h 5d41402abc4b2a76b9719d911017c592\n" +
+        "3. To crack multiple hashes from a file, use the format: ./rcrack [path to tables] -l [hash_list_file].\n" +
+        "   - Example: ./rcrack . -l hash.txt\n" +
+        "4. If you have LM hashes in a pwdump file, use the format: ./rcrack [path to tables] -lm [pwdump_file].\n" +
+        "   - Example: ./rcrack . -lm pwdump.txt\n" +
+        "5. For NTLM hashes in a pwdump file, use the format: ./rcrack [path to tables] -ntlm [pwdump_file].\n" +
+        "   - Example: ./rcrack . -ntlm pwdump.txt\n" +
+        "6. Click the 'Crack' button to execute the command and display the results."; // Steps to use the component.
```

Comment:
The format for this should be Start + Title
```suggestion
        "6. Click the Start Crack " +
        title + 
        " button to execute the command and display the results."; // Steps to use the component.
```

Reply:
I don't think its a good idea to include the title
the statement becomes
"Click the Crack RainbowCrack button to execute the command and display the results."

rather we could go for
Click the Crack button to execute the command and display the results

Reply:
Should be "Click the Start RainbowCrack button" to match our convention, but it is a minor thing, so no worries.